frontend/components/common/other/BasePanel.vue

```diff
@@ -4,6 +4,7 @@
             <div class="Icon" :class="iconClass">
                 <Icon v-if="currentState === 'inactive'" type="close" />
                 <Icon v-else-if="currentState === 'incorrect'" type="warning" theme="filled" />
+                <Icon v-else-if="currentState === 'partlyIncorrect'" type="warning" theme="filled" />
```

Comment:
I would recommend using the icon `<a-icon type="issues-close" />`. As I feel like it perfectly shows the state of it being partially correct.

![Screenshot 2022-09-29 at 17 03 14](https://user-images.githubusercontent.com/30908158/193067905-6be0b9fd-59c5-4e39-b10f-53cb2ef4655e.png)